### PR TITLE
Added supported credit card type info

### PR DESCRIPTION
**Pull request by mingazhev**
https://github.com/happy-travel/agent-app-project/issues/864

### CODE REVIEW

**Review by DFSko — APPROVED** [review]

File: HappyTravel.EdoContracts/Accommodations/Internals/CreditCardRequirement.cs [hunk 1]
```diff
@@ -22,5 +24,10 @@ public CreditCardRequirement(DateTime activationDate, DateTime dueDate)
         /// Credit card expiration date
         /// </summary>
         public DateTime DueDate { get; }
+
+        /// <summary>
+        /// Credit card types supported by the supplier. Null if all the cards are supported.
+        /// </summary>
+        public List<CreditCardTypes>? SupportedCardTypes { get; }
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Internals/CreditCardRequirement.cs:31]: May we rename it to `SupportedCardVendors`? The type is credit

Comment (mingazhev) [reply HappyTravel.EdoContracts/Accommodations/Internals/CreditCardRequirement.cs:31]: I found another definition - payment network, or payment processing network

Comment (mingazhev) [reply HappyTravel.EdoContracts/Accommodations/Internals/CreditCardRequirement.cs:31]: But in terms of code clarity, isn't `CardType` the most clear word for this?

Comment (mingazhev) [reply HappyTravel.EdoContracts/Accommodations/Internals/CreditCardRequirement.cs:31]: And I see that it is used in a such context:

https://www.valuepenguin.com/visa-vs-mastercard-is-one-better
https://www.marketwatch.com/story/visa-or-mastercardwhats-the-difference-anyway-2019-07-15

Comment (kirillta) [reply HappyTravel.EdoContracts/Accommodations/Internals/CreditCardRequirement.cs:31]: Network is better than type

Comment (mingazhev) [reply HappyTravel.EdoContracts/Accommodations/Internals/CreditCardRequirement.cs]: Renamed to CardVendor

**Review by kirillta — APPROVED** [review]

**Pull request merged**